quickwit/quickwit-storage/src/error.rs

```diff
@@ -47,6 +49,10 @@ pub enum StorageResolverError {
     /// A protocol is required for the URI.
     #[error("Invalid format for URI: required: `{message}`")]
     InvalidUri { message: String },
+    #[error("The system does not have a initialised AWS config available")]
+    /// This should never happen if the `quickwit_aws::try_init_aws_config` method
+    /// has been called, if this error occurs this is a bug.
+    MissingAWSConfig,
```

Comment:
Can we rename `MissingAWSConfig`? I think it concerns any s3 compatible storage. 
And I'm a bit confused here, the issue is not really that the config is missing right? I think we should be clearer and say something like "S3 compatible storage configuration has not been initialized. This should never happen".

Reply:
I got rid of it altogether.